x/bank/client/rest/sendtx.go

```diff
@@ -113,3 +140,240 @@ func SendRequestHandlerFn(cdc *wire.Codec, kb keys.Keybase, cliCtx context.CLICo
 		w.Write(output)
 	}
 }
+
+// RegisterSwaggerRoutes - Central function to define routes that get registered by the main application
+func RegisterSwaggerRoutes(routerGroup *gin.RouterGroup, ctx context.CLIContext, cdc *wire.Codec, kb keys.Keybase) {
+	routerGroup.POST("/accounts/:address/send", sendRequestFn(cdc, ctx, kb))
+	routerGroup.POST("/create_transfer", createTransferTxForSignFn(cdc, ctx))
+	routerGroup.POST("/signed_transfer", composeAndBroadcastSignedTransferTxFn(cdc, ctx))
```

Comment:
Would also change it to `transfers` and add a boolean parameter called `signed` in POST body. cc: @jackzampolin

Reply:
```
add a boolean parameter called signed in POST body
```
Could you clarify this? Do you mean merge /create_transfer and /signed_transfer into one rest api, such as /bank/transfers? I think this sound weird. How do you think about this? @jackzampolin

Reply:
Here we are adding to the `transfers` resource so the restful route here would be `POST /bank/transfers`. It sounds like this route has two different behaviors:

- Create a transaction formatted for signing
- Create a transaction, sign and send this transaction. 

I think the dominant behavior here would be to send a transaction and the transaction generation is more secondary. In the CLI we plan to handle this workflow (`generate`, `sign`,`send`) by adding a `--generate-only` flag to commands that normally do all 3. 

The analogue here would be to add a query param (maybe `?generate=true`) that enables the generation behavior. What do you think @fedekunze @HaoyangLiu

Reply:
Send query param in post request doesn't make sense semantically: [explanation](http://stackoverflow.com/a/26717908/418418)

I have another idea:
The below is the post body. 
```
type sendBody struct {
	Name             string    `json:"name"`
	Password         string    `json:"password"`
	Amount           sdk.Coins `json:"amount"`
	ChainID          string    `json:"chain_id"`
	AccountNumber    int64     `json:"account_number"`
	Sequence         int64     `json:"sequence"`
	Gas              int64     `json:"gas"`
	Fee		 string    `json:"fee"`
}
```
If both the two fields ("name" and "password") are not empty, then the transaction will be signed and broadcasted. If not, only generate a transaction and return the transaction bytes to user. 
What do you think about this idea? @jackzampolin @fedekunze

Reply:
I agree with the explanation above. I like your idea more but I'd propose a slight change to it, since currently `name` and `password` are also used for testing automation purposes:
```
type sendBody struct {
	Name             string    `json:"name"`
	Password         string    `json:"password"`
	Amount           sdk.Coins `json:"amount"`
	ChainID          string    `json:"chain_id"`
	AccountNumber    int64     `json:"account_number"`
	Sequence         int64     `json:"sequence"`
	Gas              int64     `json:"gas"`
	Fee		 string    `json:"fee"`
        Signed		 bool      `json:"signed"` // true by default
}
```
@jackzampolin thoughts ?